README.md

```diff
@@ -118,10 +118,11 @@ There are some alias for `optional(base)`, where base is base types, as the foll
 * `literal?(lit)`
 * `any?`
 
-Shorthands for `optional(array(ty))` and `optional(object(fields))` are also defined as the following:
+Shortcuts for complex data are also defined as the following:
```

Comment:
Use the same word in the header of this section:

> ### Shortcuts

https://github.com/soutaro/strong_json/pull/8/files#diff-04c6e90faac2675aa89e2176d2eec7d8R109